cli/scorecard/violations.go

```diff
@@ -126,35 +126,41 @@ func getViolations(inventory *InventoryConfig, config *ScoringConfig) (*validato
 
 // converts raw JSON into Asset proto
 func getAssetFromJSON(input []byte) (*validator.Asset, error) {
-	asset := tfconverter.Asset{}
+	var asset map[string]interface{}
 	err := json.Unmarshal(input, &asset)
 	if err != nil {
 		return nil, err
 	}
-
 	pbAsset := &validator.Asset{}
 	err = protoViaJSON(asset, pbAsset)
 	if err != nil {
-		return nil, errors.Wrapf(err, "converting asset %s to proto", asset.Name)
+		return nil, errors.Wrapf(err, "converting asset %s to proto", asset["Name"])
 	}
 
 	pbAsset.AncestryPath, err = getAncestryPath(pbAsset)
 	if err != nil {
-		return nil, errors.Wrapf(err, "fetching ancestry path for %s", asset.Name)
+		return nil, errors.Wrapf(err, "fetching ancestry path for %s", asset["Name"])
 	}
-
-	Log.Debug("Asset converted", "name", asset.Name, "ancestry", pbAsset.GetAncestryPath())
-
+	Log.Debug("Asset converted", "name", asset["Name"], "ancestry", pbAsset.GetAncestryPath())
 	return pbAsset, nil
 }
 
 // looks up the ancestry path for a given asset
 func getAncestryPath(pbAsset *validator.Asset) (string, error) {
```

Comment:
Could we add a link here? https://github.com/forseti-security/config-validator/blob/6cce12f5cd0e089a5371277384d9379d17b38288/pkg/asset/asset.go#L95

Reply:
this can be done after #577 is merged

Reply:
done in 61a3f4e